File: TAREA2.py
class Alumno:
    def __init__(self, nombre, edad):
        self.nombre = nombre
        self.edad = edad

alumnos = [
    Alumno("Ana", 20),
    Alumno("Luis", 22),
    Alumno("María", 19)
]

print("Lista de alumnos:")
for a in alumnos:
    print(f"{a.nombre} - {a.edad} años")

File: TAREA2.java
class Alumno {
    String nombre;
    int edad;

    Alumno(String nombre, int edad) {
        this.nombre = nombre;
        this.edad = edad;
    }
}

public class Main {
    public static void main(String[] args) {
        Alumno[] alumnos = {
            new Alumno("Ana", 20),
            new Alumno("Luis", 22),
            new Alumno("María", 19)
        };

        System.out.println("Lista de alumnos:");
        for (Alumno a : alumnos) {
            System.out.println(a.nombre + " - " + a.edad + " años");
        }
    }
}
